recipes-common/rdk-cert-config/rdkcertconfig.bb

```diff
@@ -0,0 +1,17 @@
+SUMMARY = "A simple library for certificate selector/locator"
+LICENSE = "CLOSED"
```

Comment:
Please make this Apache-2.0 in preparation for opensourcing, as this should be coming soon.